ansible.html.markdown

```diff
@@ -514,8 +514,8 @@ callback_whitelist = profile_tasks
 ```
 
 ### facts-cache and ansible-cmdb
-You can pool some infrmations of you environment from another hosts.
-If the informations does not change - you may consider using a facts_cache to speed things up.
+You can pool some information of you environment from another hosts.
```

Comment:
You can pool (or pull?) some information about your environment from other hosts.